### PR TITLE
Improve Codacy Code Quality

### PR DESCRIPTION
**Pull request by mhamilton723**

### CODE REVIEW

File: src/test/resources/benchmarks/benchmarks_VerifyLightGBMClassifier.csv [hunk 1]
```diff
@@ -1,33 +1,33 @@
 name,value,precision,higherIsBetter
-LightGBMClassifier_BreastTissue.csv_gbdt,0.9716981132075472,0.01,true
-LightGBMClassifier_BreastTissue.csv_rf,0.7358490566037735,0.01,true
-LightGBMClassifier_BreastTissue.csv_dart,0.8490566037735849,0.01,true
+LightGBMClassifier_BreastTissue.csv_gbdt,0.7924528301886793,0.01,true
```
Comment (imatiach-msft) [inline src/test/resources/benchmarks/benchmarks_VerifyLightGBMClassifier.csv]: why did the performance numbers change?

Comment (mhamilton723) [reply src/test/resources/benchmarks/benchmarks_VerifyLightGBMClassifier.csv]: lord knows

**Pull request merged**